.pre-commit-config.yaml

```diff
@@ -0,0 +1,27 @@
+exclude: '.git'
+default_stages: [commit]
+
+repos:
+  - repo: https://github.com/psf/black
+    rev: 21.12b0
+    hooks:
+      - id: black
+
+  - repo: https://github.com/timothycrosley/isort
```

Comment:
[isort](https://github.com/timothycrosley/isort) sorts imports alphabetically, and automatically separates them into sections by type.